pipeline_dp/dataset_histograms/computing_histograms.py

```diff
@@ -346,7 +470,7 @@ def _compute_l0_contributions_histogram_on_preaggregated_data(
 
     Args:
         col: collection with a pre-aggregated dataset, each element is
-        (partition_key, (count, sum, n_partitions)).
+        (partition_key, (count, sum, n_partitions, n_contributions)).
```

Comment:
Why `n_partitions` isn't always 1? if it is statistic per one partition

and why `count` is not the same as `n_contributions`?

Reply:
ah, makes sense, thank you!